mlflow/server/handlers.py

```diff
@@ -16,29 +17,42 @@
     UpdateRun, LogMetric, LogParam, SetTag, ListExperiments, \
     DeleteExperiment, RestoreExperiment, RestoreRun, DeleteRun, UpdateExperiment, LogBatch
 from mlflow.store.artifact_repository_registry import get_artifact_repository
-from mlflow.tracking.utils import _is_database_uri, _is_local_uri
+from mlflow.tracking.utils import TrackingStoreRegistry
```

Comment:
unrelated to your change -- but makes sense to move this one to `mlflow.tracking.stores.registry` package.